docs/references/rdctl-command-reference.md

```diff
@@ -0,0 +1,98 @@
+---
+title: rdctl Command Reference
+---
+
+`rdctl` is a command-line tool, included in Rancher Desktop that enables command-line access to the application features. `rdctl` is developed to help users with tasks such as scripting (for automation, CI/CD), troubleshooting, remote management etc. The current version of `rdctl` supports below commands, and support for more commands would be added in the upcoming releases.
```

Comment:
`rdctl` is a command-line tool, included in Rancher Desktop that enables command-line access to GUI features. `rdctl` is developed to help users with tasks such as scripting (for automation, CI/CD), troubleshooting, remote management, etc. The current version of `rdctl` supports the below commands (with support for more commands to be added in upcoming releases):

Reply:
Modified per your suggestion.

Reply:
I think it should mention that the current version is experimental and all subcommands names, their arguments, and their output are still subject to change.

Reply:
@jandubois , I have added a warning note that it's experimental....